app/controllers/robots_controller.rb

```diff
@@ -2,10 +2,6 @@ class RobotsController < ActionController::Base
   respond_to :text
 
   def show
-    if ENV['ALLOW_ROBOTS'].to_i == 1
-      head :ok
-    else
-      render :private_robots
-    end
+    ENV['ALLOW_ROBOTS'].to_i == 1 ? head(:ok) : render(:private_robots)
```

Comment:
I don't think ternary operators are more readable, sorry! (don't undo but just know it makes it a bit harder for me to read)

Also I think we can follow the convention of just checking if ENV is set or not.